tests/test_paper.py

```diff
@@ -0,0 +1,164 @@
+import trio
+import pytest
+import tractor
+import math
+import os
+from shutil import rmtree
+from piker.config import get_app_dir
+from piker.log import get_logger
+from piker.clearing._messages import ( 
+    Order                                    
+)
+from uuid import uuid4
+from typing import (
+    AsyncContextManager,
+    Literal,
+)
+from functools import partial
+from piker.pp import (
+    open_trade_ledger,
+    open_pps,
+)
+from piker.clearing import (
+    open_ems,
+)
+from piker.clearing._client import (
+    OrderBook,
+)
+from piker.clearing._messages import (
+    BrokerdPosition
+)
+from exceptiongroup import BaseExceptionGroup
+
+log = get_logger(__name__)
+
+@pytest.fixture(scope="session")
+def paper_cleanup():
```

Comment:
maybe add a doc string to explicitly declare the purpose of this cleanup and why we need it?